resources/spec.yaml

```diff
@@ -10,7 +10,7 @@ info:
   license:
     name: Apache-2.0
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
-  version: 1.12.0
+  version: 2.2.5
```

Comment:
Please remove this version from the spec.  It is automatically added during build

Reply:
done